src/components/TheNav.vue

```diff
@@ -1,16 +1,20 @@
+<script setup>
+import { useRouter } from 'vue-router';
+const router = useRouter();
+function goHome() {
+    router.push('/');
+}
+</script>
+
 <template>
-    <div class="ma-router-item">
-        <router-link to="/keyword-density">
-            Keyword Density
-        </router-link>
-    </div>
+    <button class="ma-btn ma-btn-secondary ma-home-fixed" @click="goHome">Home</button>
```

Comment:
you are not using the extra class in here
```suggestion
    <button class="ma-home-fixed" @click="goHome">Home</button>
```